guides/common/modules/con_overview-of-hosts-in-project.adoc

```diff
@@ -17,5 +17,9 @@ ifdef::orcharhino[]
 such as Amazon EC2, Google Compute Engine, libvirt, Microsoft Azure, Proxmox, and VMware vSphere.
 endif::[]
 
-With {Project}, you can manage hosts at scale, including monitoring, provisioning, remote execution, configuration management, software management, and subscription management.
+With {Project}, you can manage hosts at scale, including monitoring, provisioning, remote execution, configuration management,
+ifndef::foreman-deb,foreman-el[]
+subscription management,
+endif::[]
+and software management.
```

Comment:
(ouf of scope) technically correct, but the wording "software management" is quite unique! Docs usually refer to this as "content management". Your call, I am OK as is.